shard-operator/pom.xml

```diff
@@ -175,8 +175,6 @@
     <profile>
       <id>minikube</id>
       <properties>
-        <namespace>default</namespace>
-        <quarkus.kubernetes.namespace>${namespace}</quarkus.kubernetes.namespace>
```

Comment:
@vaibhavjainwiz How does this work for local-dev now? I don't see us setting this anywhere in the local environment or am I missing the configuration?

Reply:
when we run `shard-run.sh` or `mvn clean compile quarkus:dev` it uses the current namespace under `kubectl`.
```
<namespace>default</namespace>
<quarkus.kubernetes.namespace>${namespace}</quarkus.kubernetes.namespace>
```
is only for yaml generation - it sets the `namespace` in the resources generated under /target/generated-blablabla